Source/GOATracer/ViewModels/MainWindowViewModel.cs

```diff
@@ -287,19 +306,205 @@ private void DeleteLight(int lightId) {
             RequestLightUpdate?.Invoke();
         }
 
-        public async Task ImportObjFile(string filePath) {
+        /// <summary>
+        /// Helper method for printing the initial importing message
+        /// </summary>
+        /// <param name="filePath">File path to the file being imported</param>
+        private void WriteImportLogMessage(string filePath) {
             LogOutput = "";
             LogOutput += "Importing " + filePath + "...\n\n";
+        }
 
-            var sceneDescription = await Task.Run(() => ObjImporter.ImportModel(filePath));
-
+        /// <summary>
+        /// Helper method for finalizing the import (.obj or .rscn)
+        /// </summary>
+        /// <param name="sceneDescription">Scene description to be rendered</param>
+        private void FinalizeImport(ImportedSceneDescription sceneDescription) {
             if (sceneDescription != null) {
                 LoadedScene = sceneDescription;
-                LogOutput += "Import was successful!";
+                LogOutput += "Import was successful!\n";
             }
             else {
-                LogOutput += "Import failed.";
+                LogOutput += "Import failed.\n";
+            }
+        }
+
+        /// <summary>
+        /// Method for laading a RSCN format scene
```

Comment:
```suggestion
        /// Method for loading a RSCN format scene
```
Minor spelling mistake